src/OpenTelemetry.AutoInstrumentation/Configuration/ConfigurationKeys.cs

```diff
@@ -50,27 +50,27 @@ public class ConfigurationKeys
     /// <summary>
     /// Configuration key for whether the trace console exporter is enabled.
     /// </summary>
-    public const string ConsoleExporterEnabled = "OTEL_DOTNET_AUTO_TRACE_CONSOLE_EXPORTER_ENABLED";
```

Comment:
What do you think of creating a subclass `ConfigurationKeys.Tracer` and moving most/all fields into that subclass?

Reply:
Fixed bdfbbdec76779309882e3b79e17cc8fa877facea